From: Boyuan Yang <byang@debian.org>
Date: Sat, 24 Jun 2023 16:54:17 -0400
Subject: Disable test_features

The version linked with system libzstd is known to fail this test.
See https://python-zstandard.readthedocs.io/en/latest/installing.html

Patch first drafted on 2022-06-26.

Forwarded: not-needed
Last-Update: 2023-06-24
---
 tests/test_module_attributes.py | 19 -------------------
 1 file changed, 19 deletions(-)

diff --git a/tests/test_module_attributes.py b/tests/test_module_attributes.py
index dccfc27..fb8056c 100644
--- a/tests/test_module_attributes.py
+++ b/tests/test_module_attributes.py
@@ -9,25 +9,6 @@ class TestModuleAttributes(unittest.TestCase):
 
         self.assertEqual(zstd.__version__, "0.23.0")
 
-    def test_features(self):
-        self.assertIsInstance(zstd.backend_features, set)
-
-        expected = {
-            "cext": {
-                "buffer_types",
-                "multi_compress_to_buffer",
-                "multi_decompress_to_buffer",
-            },
-            "cffi": set(),
-            "rust": {
-                "buffer_types",
-                "multi_compress_to_buffer",
-                "multi_decompress_to_buffer",
-            },
-        }[zstd.backend]
-
-        self.assertEqual(zstd.backend_features, expected)
-
     def test_constants(self):
         self.assertEqual(zstd.MAX_COMPRESSION_LEVEL, 22)
         self.assertEqual(zstd.FRAME_HEADER, b"\x28\xb5\x2f\xfd")
